lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```
@@ -900,6 +914,89 @@ MITIGATION_PP_ADDRESS_AND_FRAUD:
       targetJourney: FAILED
       targetState: FAILED
 
+# Mitigation journey (05) same session - invalid-passport
+MITIGATION_05_OPTIONS:
+  response:
+    type: page
+    pageId: pyi-passport-no-match-another-way
+    mitigationStart: invalid-passport
+  events:
+    next:
+      targetState: MITIGATION_DL_CRI_DRIVING_LICENCE
+    end:
+      targetJourney: INELIGIBLE
+      targetState: INELIGIBLE_SKIP_MESSAGE
+
+# Mitigation journey (06) separate session - invalid-passport
+MITIGATION_06:
+  response:
+    type: process
+    lambda: reset-identity
+    lambdaInput:
+      isUserInitiated: false
+      deleteOnlyGPG45VCs: true
+  events:
+    next:
+      targetState: MITIGATION_06_PASSPORT_NO_MATCH_PAGE
+
+MITIGATION_06_PASSPORT_NO_MATCH_PAGE:
+  response:
+    type: page
+    pageId: pyi-passport-no-match
+  events:
+    next:
+      targetState: MITIGATION_06_IDENTITY_START_PAGE
+
+MITIGATION_06_IDENTITY_START_PAGE:
+  response:
+    type: page
+    pageId: pyi-continue-with-driving_licence
+    mitigationStart: invalid-passport
+  events:
+    next:
+      targetState: MITIGATION_DL_CRI_DRIVING_LICENCE
+    end:
+      targetJourney: INELIGIBLE
+      targetState: INELIGIBLE_SKIP_MESSAGE
+
+MITIGATION_DL_CRI_DRIVING_LICENCE:
+  response:
+    type: cri
+    criId: drivingLicence
+  parent: CRI_STATE
+  events:
+    next:
+      targetState: MITIGATION_DL_ADDRESS_AND_FRAUD
+    access-denied:
+      targetJourney: FAILED
+      targetState: FAILED
+
+MITIGATION_DL_ADDRESS_AND_FRAUD:
+  nestedJourney: ADDRESS_AND_FRAUD
+  exitEvents:
+    next:
+      targetState: MITIGATION_CHECK_FRAUD_SCORE
+    enhanced-verification:
+      targetState: MITIGATION_CHECK_FRAUD_SCORE
```

This isn't an event we should ever see from this state, but we should probably send it to `FAILED` rather than continue mitigating

---

done